modules/clair-openshift-config.adoc

```diff
@@ -1,43 +1,35 @@
-[[clair-openshift-config]]
-= Clair on OpenShift config
+// Module included in the following assemblies:
+//
+// clair/master.adoc
 
-To retrieve the configuration file for a Clair instance deployed using the OpenShift Operator, retrieve and decode the config secret using the appropriate namespace, and save it to file, for example:
+:_content-type: PROCEDURE
+[id="clair-openshift-config"]
+= Retrieving and decoding the Clair configuration secret for Clair deployments on {ocp}
```

Comment:
This section feels like it ends abruptly. It does what it says, getting and decoding the config, but then doesn't point to how the config can be used. Maybe just add a final blurb or step about what to do with it after decoding?